src/components/Player/Player.tsx

```diff
@@ -161,6 +172,23 @@ const Player: React.FC<Props> = ({
 
       playerRef.current = window.jwplayer(playerElementRef.current) as JWPlayer;
 
+      // Add user_id and profile_id for CDN analytics
+      const { sources } = item;
+
+      sources?.map((source) => {
+        const url = new URL(source.file);
+        const isVOD = url.pathname.includes('manifests');
+        const isBCL = url.pathname.includes('broadcast');
+        const isDRM = url.searchParams.has('exp') || url.searchParams.has('sig');
```

Comment:
Should we use the app config drm property for this instead of this check which may be a bit flaky?